policy/README.md

```diff
@@ -0,0 +1,353 @@
+2. Providers, so that they can measure their compliance status independently
+3. Tooling companies that want to create applications for policy-creation and editing
+
+<a name="background"></a>
+## Background
+
+The goal of this specification is to enable Agencies to create, revise, and publish machine-readable micromobility policy, as set of rules for individual and collective micromobility device behavior exhibited by both providers and riders. Examples of policies include:
+
+* City-wide and localized caps (e.g. “Minimum 500 and maximum 3000 scooters within city boundaries”)
+* Exclusion zones (e.g. “No scooters are permitted in this district on weekends”)
+* Cap allowances (e.g. “Up to 500 additional scooters are permitted near train stations”)
+* Speed-limit restrictions (e.g. “15 mph outside of downtown, 10 mph downtown”)
+* Idle-time and disabled-time limitations (e.g. “5 days idle while rentable, 12 hours idle while unrentable, per device”)
+
+A machine-readable format will allow Providers to download policies and compute compliance for policies where it can be determined entirely by data obtained internally.  Providers can then continually measure their own compliance against policies without further API calls.
+
+References to geography (areas, street segments, etc.) will be done via UUID.  Geographic data will be available as GeoJSON via the `/geographies` endpoint.  In a future revision of Agency, we will reconcile this with the existing `/service_areas` endpoint.  `/service_areas` currently only handles GeoJSON MultiPolygon and Polygon objects, and Policies might refer to street segments or points.  
+
+This initial draft proposes a subset of possible policies for consideration, and should not be taken to be the a comprehensive enumeration of all possible policies.
+
+<a name="distribution"></a>
+## Distribution
+
+Policies may be published by Agencies or their authorized delegates via JSON objects via REST API.
+
+Each policy will have a unique ID (UUID).
+
+Published policies should be treated as immutable data.  Obsoleting or otherwise changing policies is accomplished by publishing additional policies with a field named `prev_policies`, a list of UUID references to the previous policy or policies.  
+
+Policies should be stored and accessible indefinitely so that the set of active policies at a given time in the past can be retrieved from the `/policies` endpoint.
+
+Policies will typically be linked to one or more associated geographies.  Geography descriptions (e.g. geofences or lists of street segments) must also be maintained by the Agency indefinitely.  Policies without specific geographies (global policies) are assumed to apply to the entire service area managed by the Agency.
+
+Policies should be re-fetched whenever (a) a Policy expires (via its `end_date`), or (b) at an interval specified by the Agency, e.g. "daily at midnight".
+
+
+<a name="schema"></a>
+## Schema
```

Comment:
We shouldn't merge this until we have a JSON schema, so that is easy to build "policy" validator tools. We might even want to include a script to validate policies in the spec more than just JSON schema. More work I know but I think it is really important that we make sure there is no way to publish a valid policy that can be misinterpreted.

Reply:
Roger

Reply:
Schema added